src/styles/index.scss

```diff
@@ -1,3 +1,153 @@
+$font-family-primary: 'Roboto', Arial, sans-serif;
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: 400px;
```

Comment:
```suggestion
  font-weight: 400;
```
fix it:
![image](https://github.com/user-attachments/assets/4332c0c4-3c49-4847-96b3-a358038b8b07)